llama-index-integrations/embeddings/llama-index-embeddings-alibabacloud-aisearch/llama_index/embeddings/alibabacloud_aisearch/base.py

```diff
@@ -0,0 +1,182 @@
+import asyncio
+import time
+from typing import Any, List
+
+from llama_index.core.base.embeddings.base import BaseEmbedding
+from llama_index.core.bridge.pydantic import Field, PrivateAttr
+
+
+from llama_index.core.bridge.pydantic import Field, PrivateAttr
+
+from llama_index.core.base.llms.generic_utils import get_from_param_or_env
+
+try:
+    from alibabacloud_searchplat20240529.models import (
+        GetTextEmbeddingRequest,
+        GetTextEmbeddingResponse,
+    )
+    from alibabacloud_tea_openapi.models import Config as AISearchConfig
+    from alibabacloud_searchplat20240529.client import Client
+    from Tea.exceptions import TeaException
+except ImportError:
+    raise ImportError(
+        "Could not import alibabacloud_searchplat20240529 python package. "
+        "Please install it with `pip install alibabacloud_searchplat20240529`."
```

Comment:
Is that actually the name of the package? Thats wild haha

Reply:
Yes, it is. haha